docs/consume-signal.md

```diff
@@ -0,0 +1,117 @@
+# Consume cerberus signal
+
+Various examples on how to comsume the Cerberus signal :
+
+- Using a simple http client (cURL)
+- As an init container for a Pod
+- As a tekton task (part of a pipeline)
+
+## Simple check with curl
+
+if you just want to check the cerberus signal, you can use this small script to check the cerberus signal.
+
+```bash
+export CERBERUS_URL=http://cerberus.chaos-cerberus.svc.cluster.local:8080
+if curl -s "$CERBERUS_URL" | grep True &> /dev/null; then
+    echo "Cerberus check is OK at ${CERBERUS_URL}"
+else
+    echo "Cerberus check is NOT OK at ${CERBERUS_URL}"
+fi
+```
+
+## As an init container
+
+This example allow you to start a container when cerberus return a OK signal.
+
+```yaml
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cerberus-config-job
+data:
+  cerberus_url: "cerberus.default.svc.cluster.local" 
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "check-node-if-cerberus-ok"
+spec:
+  template:
+    spec:
+      initContainers:
+      - name: check-cerberus
+        image: quay.io/startx/runner-oc:fc35
+        command: ['bash', '-c', "until curl -s $CERBERUS_URL | grep True &> /dev/null; do echo Wait for OK from cerberus at $CERBERUS_URL; sleep 2; done"]
+        env:
+        - name: CERBERUS_URL
+          valueFrom:
+            configMapKeyRef:
+              name: cerberus-config-job
+              key: cerberus_url
+      containers:
+      - name: job-task
+        image: "quay.io/startx/runner-oc:fc35"
+        command:
+          - "/bin/bash"
+          - "-c"
+          - |-
+            echo "executed after a POSITIVE cerberus check agains't $CERBERUS_URL"
+            echo "Replace this container definition with your own description"
+            exit
+        env:
+        - name: CERBERUS_URL
+          valueFrom:
+            configMapKeyRef:
+              name: cerberus-config-job
+              key: cerberus_url
+        resources:
+          requests:
+            cpu: "10m"
+            memory: "64Mi"
+          limits:
+            cpu: "50m"
+            memory: "128Mi"
+      restartPolicy: Never
+  backoffLimit: 2
+```
+
+## Using tekton pipeline
```

Comment:
@startx-lab thoughts on creating a new section called use cases/Cerberus in practice and moving the content over there similar to how you did for Kraken: https://github.com/redhat-chaos/krkn/pull/333?